packages/medusa/src/config/index.js

```diff
@@ -26,6 +26,9 @@ const config = {
   cookieSecret:
     process.env.NODE_ENV === "test" ? "test" : process.env.COOKIE_SECRET,
 
+  // Max Age for Session Cookie. 10 Hours by default
```

Comment:
My only nit-pick would be to consider changing the comment to use the same format as the other comments, just to keep the convention.

E.g. 
```
/**
* Max Age for Session Cookie. 10 Hours by default
*/
```